packages/create-sitecore-jss/src/templates/nextjs/scripts/templates/component-factory.ts

```diff
@@ -52,7 +52,8 @@ ${componentFiles
       const moduleName = removeDynamicModuleNameEnding(component.moduleName);
       return `const ${moduleName} = {
   module: () => import('${component.path}'),
```

Comment:
And can you try to use `module: () => import('${component.path}'),` as it was before, instead of `require()` is it working?
I bet that `import()` is async